client/my-sites/preview/main.jsx

```diff
@@ -119,6 +119,11 @@ class PreviewMain extends React.Component {
 					showClose={ false }
 					previewUrl={ this.state.previewUrl }
 					externalUrl={ this.state.externalUrl }
+					loadingMessage={
+						this.props.translate( '{{strong}}One moment please…{{/strong}} loading your site.',
```

Comment:
I know nothing about how `a8ci18n` works, but it feels like we could have a set of keywords ('post', 'site', 'project' etc.) that are recognised as not being interchangeable - the bot would see that the difference is these 2 words and know not to offer a suggestion.
Though, the above is still useful even just for making sure the formatting of the sentence is the same...